core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -893,14 +896,29 @@ class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient,
    * @return list of child node names
    */
   def getChildren(path : String): Seq[String] = {
-    val getChildrenResponse = retryRequestUntilConnected(GetChildrenRequest(path, registerWatch = true))
+    val getChildrenResponse = retryRequestUntilConnected(
```

Comment:
nit:  Let's match the style of the parenthesis/bracelet
1. `if` block has the bracelets, so let's have it on else too
2. `(` follows a line break, so let's break the line before the `)` too
i.e. either
```scala
  def getChildren(path : String): Seq[String] = {
    val getChildrenResponse = retryRequestUntilConnected(
      if (shouldPaginate(path)) {
        debug(s"upgrading GetChildrenRequest to GetChildrenPaginatedRequest for '${path}'")
        GetChildrenPaginatedRequest(path, registerWatch = true)
      } else {
        GetChildrenRequest(path, registerWatch = true)
      }
    )
    getChildrenResponse.resultCode match {
      case Code.OK => getChildrenResponse.children
      ...
```
or
```scala
  def getChildren(path : String): Seq[String] = {
    val getChildrenResponse = retryRequestUntilConnected(if (shouldPaginate(path)) {
                                                           debug(s"upgrading GetChildrenRequest to GetChildrenPaginatedRequest for '${path}'")
                                                           GetChildrenPaginatedRequest(path, registerWatch = true)
                                                         } else {
                                                           GetChildrenRequest(path, registerWatch = true)
                                                         })
    getChildrenResponse.resultCode match {
      case Code.OK => getChildrenResponse.children
      ...
```

I personally prefer the latter as it makes it clear that the if block is a ternary operator that evaluates to a value, but both are fine since they're all consistent.